packages/pilot/public/locales/pt/translations.json

```
@@ -942,10 +944,26 @@
       "wrong_password": "Senha inválida"
     },
     "reprocess": {
-      "submit": "Reprocessar transação",
+      "action_confirm": "Tem certeza que deseja reprocessar essa transação?",
       "confirmation": "Confirmação",
+      "reprocess": "Reprocessar transação",
+      "reprocess_with_antifraud": "Reprocessar com antifraude",
+      "reprocess_without_antifraud": "Reprocessar sem antifraude",
       "result": "Conclusão",
       "success": "A transação foi reprocessada com sucesso",
+      "type_selection": "Como você deseja reprocessar essa transação?",
+      "without_antifraud_disclaimer_1": "Ao prosseguir com a opção ",
+      "without_antifraud_disclaimer_2": "sem antifraude",
+      "without_antifraud_disclaimer_3": ", você reprocessará uma transação com potencial risco de fraude e as consequências decorrentes da decisão serão de sua integral responsabilidade. Leia mais sobre reprocessamento na ",
+      "without_antifraud_disclaimer_4": "nossa documentação.",
+      "without_antifraud_documentation_link": "https://docs.pagar.me/page/reprocessamento-sem-antifraude",
+      "confirmation_reprocess_question": "Você está reprocessando uma transação de alto risco, deseja continuar?",
+      "confirmation_with_antifraud_title": "Não se esqueça de entrar em contato com o comprador",
+      "confirmation_with_antifraud_description": "Antes de solicitar um reprocessamento, é indispensável que você valide seus dados do cartão, endereço de entrega e veracidade da transação do seu cliente.",
```

Há algumas diferenças aqui em relação ao zeplin:

```suggestion
      "confirmation_with_antifraud_description": "Antes de solicitar um reprocessamento, é indispensável que você valide os dados do cartão, endereço de entrega e veracidade da transação com o seu cliente.",
```

---

Resolvido!